### ISSUE TITLE
"Floating point exception" after trying the method from the issue #22382

### ISSUE THREAD

**Issue by XXWong** [pytorch/pytorch#22487]
in order to use libtorch c++ API.I tried an example from:https://pytorch.org/tutorials/advanced/cpp_export.html#a-minimal-c-application.
Firstly, I got the same bug report with issue #22382.
Then, I tried the method from it and 'make' succeed!
Howerver, I got the problem 'Floating point exception' after executing './example model.pt'
I debug and find the problem was caused by the code 'at::Tensor output = module.forward(inputs).toTensor()'
and I don't know why?

**Comment by izdeby** [comment]
Hi, @XXWong, could you, please, post the exception message?

**Comment by XXWong** [comment]
**example.cpp**

#include <torch/script.h> // One-stop header.
#include <iostream>
#include <memory>

int main(int argc, const char* argv[]) {
  if (argc != 2) {
    std::cerr << "usage: example-app <path-to-exported-script-module>\n";
    return -1;
  }

  // Deserialize the ScriptModule from a file using torch::jit::load().
  torch::jit::script::Module module = torch::jit::load(argv[1]);
  std::cout << "ok\n";
  // Create a vector of inputs.
  std::vector<torch::jit::IValue> inputs;
  inputs.push_back(torch::ones({1, 3, 224, 224}));
  
  // Execute the model and turn its output into a tensor.
  at::Tensor output = module.forward(inputs).toTensor();
  std::cout << output.slice(/*dim=*/1, /*start=*/0, /*end=*/5) << '\n';
}

**After executing "./example model.pt"
I just get the output with "Floating point exception" 

Moreover, if I annotate the lines "  
at::Tensor output = module.forward(inputs).toTensor();
std::cout << output.slice(/*dim=*/1, /*start=*/0, /*end=*/5) << '\n';
" 
I can get the correct output "ok"
So I wonder that the error has something to do with the usage of "module.forward(inputs)"**

**Comment by janeyx99** [comment]
TorchScript has been deprecated in favor of torch.export as of 2.11. https://pytorch.org/blog/pytorch-2-11-release-blog/ Closing this issue as we are not further developing on it--please ping if you believe this issue was erroneously closed.
